qmcpy/algorithms/function/asian_call_integrand.py

```diff
@@ -0,0 +1,50 @@
+''' Originally developed in MATLAB by Fred Hickernell. Translated to python by Sou-Cheng T. Choi and Aleksei Sorokin '''
+import numpy as np
+
+from algorithms.function.integrand_base import IntegrandBase
+
+
+class AsianCallFun(IntegrandBase):
+    """
+    Specify and generate payoff values of an Asian Call option
+
+    TODO - Come up with better, more interpretable, lowercase names for these parameters
+
+    """
+    def __init__(self, bm_measure=None, volatility=.5, S0=30, K=25, nominal_value=None):
```

Comment:
I mentioned this somewhere else, but variable names should be intelligible and lowercase.  This is math code, so some extra flex is permissible (such as just using the letter `K` rather than saying `kernel_matrix`), but not as arguments in a function (only within the function as a local variable).

Reply:
These have been changed in the most recent commit. 
s0 -> initial_price
k -> strike_price